apps/dashboard/src/main/java/com/akto/action/testing/StartTestAction.java

```diff
@@ -315,6 +319,8 @@ public String retrieveAllCollectionTests() {
 
         if(fetchCicd){
             testingRunFilters.add(Filters.in(Constants.ID, getCicdTests()));
+        } else if(fetchAll){
+            testingRunFilters.add(Filters.ne("triggeredBy", "test_editor"));
```

Comment:
fix the naming. Doesn't give any clue on what this is supposed to do

Reply:
done